backend/src/errors.ts

```diff
@@ -19,7 +53,21 @@ export enum ErrorMessages {
     GetAllItems = "Error while retrieving all items:"
 }
 
-export const sendError = (res: Response, errorMessage: ErrorMessages, err: any) => {
-    console.error(errorMessage, err);
-    res.status(500).json({error: errorMessage});
+export const sendError = (res: Response, err: CustomError) => {
+    console.error(err.typology, err);
+
+    if (err instanceof ValidationError) {
```

Comment:
issue(non-blocking) : use early return pattern to avoid if .. else ifs else.

```
if(...) {
  res.Status(400).json(...)
   return
}
if(...)
  res.status(404).json(...)
  return
```